neqo-transport/src/tracking.rs

```diff
@@ -48,8 +48,8 @@ impl PacketNumberSpace {
 impl From<Epoch> for PacketNumberSpace {
     fn from(epoch: Epoch) -> Self {
         match epoch {
-            TLS_EPOCH_INITIAL => Self::Initial,
-            TLS_EPOCH_HANDSHAKE => Self::Handshake,
+            Epoch::Initial => Self::Initial,
+            Epoch::Handshake => Self::Handshake,
             _ => Self::ApplicationData,
```

Comment:
```suggestion
            Epoch::ApplicationData | Epoch::ZeroRtt => Self::ApplicationData,
```

The underscore doesn't make sense any more.  Better to spell it out.

It might be worth documenting that we don't have a 0-RTT packet number space in the enum.  The basic story is that we use the application data space for that, using the appropriate keys based on availability.  (`CryptoSpace` holds the analogous values to `Epoch`.  Which leads me to the next comment...